hr_timesheet_begin_end/tests/test_timesheet_begin_end.py

```diff
@@ -22,18 +22,20 @@ def setUp(self):
             "amount": -60.0,
         }
 
-    def test_onchange(self):
+    def test_compute_unit_amount(self):
         line = self.timesheet_line_model.new(
             {"name": "test", "time_start": 10.0, "time_stop": 12.0}
         )
-        line.onchange_hours_start_stop()
         self.assertEqual(line.unit_amount, 2)
+        line.time_stop = 14.0
+        self.assertEqual(line.unit_amount, 4)
 
-    def test_onchange_no_update(self):
+    def test_compute_unit_amount_no_update(self):
```

Comment:
this should be renamed, as it actually updates the value (sets it to 0) (even though it was already at 0).

Reply:
Done in #692